cmd/neofs-node/config/node/config.go

```diff
@@ -24,16 +27,40 @@ var (
 // from "node" section.
 //
 // Panics if value is not a non-empty string.
-func Key(c *config.Config) string {
+func Key(c *config.Config) *keys.PrivateKey {
 	v := config.StringSafe(c.Sub(subsection), "key")
 	if v == "" {
 		panic(errKeyNotSet)
 	}
 
-	// TODO: add string -> `ecdsa.PrivateKey` parsing logic
-	// after https://github.com/nspcc-dev/neofs-node/pull/569.
+	var (
+		key  *keys.PrivateKey
+		err  error
+		data []byte
+	)
+	if data, err = ioutil.ReadFile(v); err == nil {
+		key, err = keys.NewPrivateKeyFromBytes(data)
+	}
+
+	if err != nil {
+		panic(fmt.Errorf("can't read key: %w", err))
+	}
+
+	return key
+}
+
+// Wallet returns value of node private key from "node" section.
+func Wallet(c *config.Config) *keys.PrivateKey {
+	v := c.Sub(subsection).Sub("wallet")
+	acc, err := innerringconfig.LoadAccount(
```

Comment:
This looks really odd to import something from innerring config to storage node. Can we move it somewhere in `pkg`? `pkg/util` or some sort.